### ISSUE TITLE
instragram, deviantart and flickr fail on reddit?

### ISSUE THREAD

**Issue by openffchrome** [RipMeApp/ripme#1052]
* Ripme version: 1.7.69
* Java version: 1.8.0
* Operating system Win10
* Exact URL you were trying to rip when the problem occurred: multiples url
* Please include any additional information about how to reproduce the problem:

instragram, flicker, deviantart single images are both failing using rippme on redddit sub, is this normal?

> 2018-11-15 21:44:05,611 DEBUG utils.RipUtils .getFilesFromURL() Checking https://www.flickr.com/photos/oneterry/38523987320/in/dateposted/
> 2018-11-15 21:44:05,612 ERROR utils.RipUtils .getFilesFromURL() [!] Unable to rip URL: https://www.flickr.com/photos/oneterry/38523987320/in/dateposted/
> 2018-11-15 21:44:21,044 DEBUG utils.RipUtils .getFilesFromURL() Checking https://www.flickr.com/photos/_ad/26265248329
> 2018-11-15 21:44:21,044 ERROR utils.RipUtils .getFilesFromURL() [!] Unable to rip URL: https://www.flickr.com/photos/_ad/26265248329

> 2018-11-15 21:44:24,507 DEBUG utils.RipUtils .getFilesFromURL() Checking https://www.instagram.com/p/BYNq5zfD7mL/
> 2018-11-15 21:44:24,507 ERROR utils.RipUtils .getFilesFromURL() [!] Unable to rip URL: https://www.instagram.com/p/BYNq5zfD7mL/
> 2018-11-15 21:41:56,616 DEBUG utils.RipUtils .getFilesFromURL() Checking https://www.instagram.com/p/Bp-Yoo6Awha/?utm_source=ig_share_sheet&amp;igshid=qfhyj3rqwpwi
> 2018-11-15 21:41:56,616 ERROR utils.RipUtils .getFilesFromURL() [!] Unable to rip URL: https://www.instagram.com/p/Bp-Yoo6Awha/?utm_source=ig_share_sheet&amp;igshid=qfhyj3rqwpwi

> 2018-11-16 09:03:20,024 DEBUG utils.RipUtils .getFilesFromURL() Checking http://505cf.deviantart.com/art/Strawberry-Heart-600986329
> 2018-11-16 09:03:20,025 ERROR utils.RipUtils .getFilesFromURL() [!] Unable to rip URL: http://505cf.deviantart.com/art/Strawberry-Heart-600986329
> 2018-11-16 09:21:30,193 DEBUG utils.RipUtils .getFilesFromURL() Checking http://extremeprejudice.deviantart.com/art/The-107th-Floor-2016-05-05-001-665544701
> 2018-11-16 09:21:30,193 ERROR utils.RipUtils .getFilesFromURL() [!] Unable to rip URL: http://extremeprejudice.deviantart.com/art/The-107th-Floor-2016-05-05-001-665544701

**Comment by cyian-1756** [comment]
> instragram, flicker, deviantart single images are both failing using rippme on redddit sub, is this normal?

Yes. Current the reddit ripper doen't support ripping from instragram, flicker or deviantart
